tests/integration/bond_test.py

```diff
@@ -1012,3 +1013,21 @@ def test_create_bond_with_copy_mac_from(eth1_up, eth2_up):
     ):
         current_state = statelib.show_only((BOND99,))
         assert_mac_address(current_state, eth2_mac)
+
+
+@pytest.mark.xfail(
+    nm_major_minor_version() < 1.30,
+    reason="Before 1.30 the bond was not keeping the mac",
+)
+def test_replacing_port_do_not_keep_mac_of_bond(bond99_with_eth2, eth1_up):
```

Comment:
The test name should describe what is expected here. The current name contradicts the reason, too.

Also, IMHO, it would be good to link the test to a BZ and make somehow else sure that everyone can easily understand in a year why we have this test.

Reply:
The reason was incorrect. Fixed. I've included the BZ in the reason. I think that is enough to understand in a year why we have this test as the BZ contains all the details of the issue.